### ISSUE TITLE
Where is SDFX UI?

### ISSUE THREAD

**Issue by Tobe2d** [sdfxai/SDFXBridgeForComfyUI#1]
So I installed the SDFXBridgeForComfyUI following the documentations however, where to see the UI for SDFX?

this step is missing in the instructions ;))

**Comment by kevin96666** [comment]
Dear. Please find sdfx here : 
https://github.com/sdfxai/sdfx

**Comment by Tobe2d** [comment]
Thnaks for the reply, So I still need to install SDFX itself?
As wheen I try it did install comfyui while I do have it so I had to stop it.
just need some guide on this point please.

**Comment by kevin96666** [comment]
sdfx has 2 install process : 
1) you don't have ComfyUI so SDFX installer will do everything for you including installing ComfyUI, and our custom_node SDFXBridgeForComfyUI
2) you already have ComfyUI installed and installed SDFXBridgeForComfyUI so you just want to use sdfx in standalone mode : 
      => in this case you just need to follow the documentation here : https://github.com/sdfxai/sdfx?tab=readme-ov-file#installation-for-users-already-using-comfyui-locally
      In a nutshell you just need to cd src && npm install && npm run start on sdfx without using the installer

**Comment by Tobe2d** [comment]
I am so sory but the instructions are not clear. 

every few steps it refear me to another link and I get lost with so many instructions not concestant.

I would expect the instructions to be much more clear like for example:

Installation for users already using ComfyUI Locally
 1 -  install the custom_node using comfy manager or git clone https://github.com/sdfxai/SDFXBridgeForComfyUI.git
in `ComfyUI\custom_nodes`
 `cd SDFXBridgeForComfyUI` 
 and `pip install -r requirements.txt`
 
  2 - Download [SDFX](https://github.com/sdfxai/sdfx) and install.
```
  git clone https://github.com/sdfxai/sdfx.git
  cd sdfx
```
Then open `setup.bat` to install dependencies and follow the steps in terminal to select GPU or CPU ...

 3 - locate sdfx.config.json in SDFX root folder and change the path to your original comfyui paths forall of them.
optinal: you may change port number if needed in  locate sdfx.config.json

  4 - run.bat

-------------------------

Few notes:
in regards to  sdfx.config.json
I edited the paths but it was really hard to add the absolute path for each entry and I had to change "\" to "/" as I am on windows it was not detecting it and I had two options eather use dubble \\ or single / and some of the m was not even clear how to connect to original models folder in comfyui as standard practice like for example upscale_models as we normaly use GFPGAN or other ones to upscale. the same goes for style_models

Hope the documentations will be clearer for normal people as for me I find myways through this as I am almost day one comfyui user and I know how to navigate through files and folders 

the instructions steps above I just made it as easy example for users as people will be lost to keep jumping  from page to page finding steps.


Hope this will help you gus as I think the project should be super suceesful as it looks really awesome!
yet I have to play with it for sometime to decicde how usefull it is ;-)



after all the above i got this errror:

![image](https://github.com/sdfxai/SDFXBridgeForComfyUI/assets/4099839/c3c7cdef-87e1-42f7-9153-4b986b574a36)
using this json: 
[timeline.sd15.sdfx.json](https://github.com/sdfxai/SDFXBridgeForComfyUI/files/14959532/timeline.sd15.sdfx.json)

**Comment by kevin96666** [comment]
Thanks for your feedback, awesome.
Regarding the SyntaxError please refresh the app, Everything is still in beta.
Have a good day.
